.semaphore/semaphore.yml

```diff
@@ -65,7 +65,7 @@ after_pipeline:
           - test-results gen-pipeline-report
 
 promotions:
-  - name: Deploy
-    pipeline_file: deploy.yml
+  - name: Docker build and push
+    pipeline_file: docker.yml
     auto_promote:
-      when: (branch = 'master' AND result = 'passed') OR (tag =~ '^release-.*' AND result = 'passed')
+      when: (branch = 'master' AND result = 'passed') OR (tag =~ '^release-.*' AND result = 'passed') OR (branch = 'containerization')
```

Comment:
We can't replace this wholesale because it'll break existing deployments, right?

Reply:
Added it for debugging build and deployment. Removed it